senior_project_dea-main/server/controllers/games.js

```diff
@@ -21,25 +21,6 @@ const getGameCount = (async(req,res) =>{
     });
 })
 
-const getAllGamesCount = (async(req,res) =>{
```

Comment:
realized this was not needed since we're linking CYOA and DND questions to GameQuestion records